src/packet_analysis/protocol/tcp/TCPSessionAdapter.cc

```diff
@@ -379,7 +381,7 @@ static zeek::RecordValPtr build_syn_packet_val(bool is_orig, const zeek::IP_Hdr*
 			if ( opt_len < 4 )
 				break;	// bad length
 
-			MSS = (options[2] << 8) | options[3];
+			MSS = ntohs(*reinterpret_cast<const uint16_t*>(options + 2));
```

Comment:
This looks like an unrelate change? I also believe the original version was done that way on purpose because the `options` pointer might not be guaranteed to be aligned correctly.